go.mod

```diff
@@ -253,6 +253,9 @@ require (
 )
 
 replace (
+	// TODO release gitops-engine and remove the line bellow
+	github.com/argoproj/gitops-engine => github.com/leoluz/gitops-engine v0.4.1-0.20220725160058-77142a1d6e59
```

Comment:
This can be updated now right? https://github.com/argoproj/gitops-engine/pull/418

quite excited for this thanks for the awesome work @leoluz

Reply:
@FabioAntunes Tks!
I just updated this branch to point to gitops-engine master branch.
It would be awesome if you can give this a try and provide us feedback.